src/coprocessor/codec/batch/lazy_column_vec.rs

```diff
@@ -317,84 +315,85 @@ mod tests {
             },
         ];
 
-        let mut rows = LazyBatchColumnVec::<()>::raw(2, 3);
-        assert_eq!(rows.rows_len(), 0);
-        assert_eq!(rows.columns_len(), 2);
-        assert_eq!(rows.rows_capacity(), 3);
-        rows.retain_by_index(|_| true);
-        assert_eq!(rows.rows_len(), 0);
-        assert_eq!(rows.columns_len(), 2);
-        assert_eq!(rows.rows_capacity(), 3);
-        rows.retain_by_index(|_| false);
-        assert_eq!(rows.rows_len(), 0);
-        assert_eq!(rows.columns_len(), 2);
-        assert_eq!(rows.rows_capacity(), 3);
-
-        rows.push_raw_row(raw_row_from_datums(&[None, Some(Datum::F64(1.3))], false));
-        rows.push_raw_row(raw_row_from_datums(&[None, None], false));
-        rows.push_raw_row(raw_row_from_datums(&[Some(Datum::U64(3)), None], true));
-        rows.push_raw_row(raw_row_from_datums(
+        let mut columns = LazyBatchColumnVec::raw(2, 3);
```

Comment:
maybe we could use `table` instead of `columns`?

Reply:
I prefer `columns` to be more clear because it's just a vector of columns.

Reply:
OK